### PR TITLE
school library set

### PR DESCRIPTION
**Pull request by DavyKn**
~Implementation of classes and objects in Ruby.
~Implementation of  encapsulation and inheritance with Ruby.
~correct gitflow

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Person
+  def initialize(_age, name = 'unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @parent_permission = parent_permission
+  end
```
Comment (mikethreels) [inline person.rb:7]: - [x] there should be an instance variable for age as well.

File: person.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+  def can_use_services?
+    of_age? || @parent_permission ? true : false
+  end
+
+  private
+
+  def of_age?
+    @age <= 18
+  end
```
Comment (mikethreels) [inline person.rb]: - [x] the condition should be if @age is larger than 18 instead of smaller than.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Katana, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by DavyKn** [comment]
all requested changes have been made

**Review by DeliceLydia — APPROVED** [review]
## Hi @DavyKn, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**